py/selenium/webdriver/remote/remote_connection.py

```diff
@@ -36,6 +36,11 @@
 LOGGER = logging.getLogger(__name__)
 
 remote_commands = {
+    Command.ADD_INTERCEPT: ("POST", "/session/$sessionId/network/intercept"),
```

Comment:
We don't need those, do we?

Reply:
I thought we might, but I'll defer to you